controller/src/main/java/io/pravega/controller/server/rpc/auth/PasswordAuthHandler.java

```diff
@@ -11,8 +11,8 @@
 
 import com.google.common.base.Preconditions;
 import com.google.common.base.Strings;
-import io.pravega.auth.PravegaAuthHandler;
-import io.pravega.auth.PravegaAuthenticationException;
+import io.pravega.auth.AuthHandler;
```

Comment:
I can't seem to find where `AuthHandler` now resides. (the actual file).

Also it seems the branch doesn't build at the momebnt:
* Where:
Build file '/Users/cbalczun/DEV/currentdev/nautilus/repos/pravega/build.gradle' line: 98

* What went wrong:
A problem occurred evaluating root project 'pravega'.
> Could not find method javadoc() for arguments [build_bq1zbia6udtm8dl3qqj1jbcu0$_run_closure4$_closure42@694e2f53] on project ':authplugin' of type org.gradle.api.Project.

Reply:
Hi @sarlaccpit I am working on recent CR comments. There is still going to be some changes in next couple of days. I recommend not trying out the latest changes till the branch is stable. Will let you know once the branch is stable. Apologies for not doing so earlier.

Reply:
got it. no problem.